en/components/toc.yml

```diff
@@ -154,24 +154,24 @@
   href: excel_library.md
   new: true
   items:
-    - name: Using Cells
-      href: excel_library_using_cells.md
-      new: true
-    - name: Using Tables
-      href: excel_library_using_tables.md
-      new: true
-    - name: Using Workbooks
-      href: excel_library_using_workbooks.md
-      new: true
-    - name: Using Worksheets
-      href: excel_library_using_worksheets.md
-      new: true
+    # - name: Using Cells
+    #   href: excel_library_using_cells.md
+    #   new: true
+    # - name: Using Tables
+    #   href: excel_library_using_tables.md
+    #   new: true
+    # - name: Using Workbooks
+    #   href: excel_library_using_workbooks.md
+    #   new: true
+    # - name: Using Worksheets
+    #   href: excel_library_using_worksheets.md
+    #   new: true
     - name: Working with Charts
```

Comment:
suggest using these titles for topics:
- Workbook Operations
- Worksheet Operations
- Working with Cells
- Working with Tables

Reply:
ok, I'll fix this when we enable them back